### PR TITLE
Minimalist playbook to view info from vSphere

### PR DESCRIPTION
**Pull request by acozine**
This playbook returns information about:
- templates in vSphere
- hosts in vSphere
- datastores in vSphere
Eventually, this will support creating VMs from Tower.

For now, the playbook only gathers information. We can add more active tasks to it in a branch once this one is merged to main.

### CODE REVIEW

File: playbooks/utils/vSphere_audit.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        show_cluster: true
+        show_esxi_hostname: true
+        vm_type: template
+      register: template_info
+
+    - name: View template info
+      ansible.builtin.debug:
+        var: template_info
+
+    - name: Gather info on hosts
```
Comment (kayiwa) [inline playbooks/utils/vSphere_audit.yml:26]: can we distinguish between ESXi host and host. I think we are asking for the ESXi host but not sure.

File: playbooks/utils/vSphere_audit.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        password: "{{ vcenter_password }}"
+        validate_certs: false
+        show_datacenter: true
+        # schema: vsphere
+        # properties:
+        #   - hardware.memorySize
+        #   - something like numberOfVMs?
+      register: host_info
+
+    - name: View host info
```
Comment (kayiwa) [inline playbooks/utils/vSphere_audit.yml:39]: Same here. I am unsure but can guess this is ESXi

**Review by kayiwa — APPROVED** [review]

File: playbooks/utils/vSphere_audit.yml (hunk 1 above)
Comment (acozine) [reply playbooks/utils/vSphere_audit.yml]: Good idea, thanks. Addressed in c1e12912b6ff1ba3f55d0bfb0f37a5e98ce8f3b5.

File: playbooks/utils/vSphere_audit.yml (hunk 2 above)
Comment (acozine) [reply playbooks/utils/vSphere_audit.yml]: Thanks again. Addressed in c1e12912b6ff1ba3f55d0bfb0f37a5e98ce8f3b5.

**Pull request merged**